### PR TITLE
Fix OCS-CI container

### PR DESCRIPTION
**Pull request by OdedViner**

### CODE REVIEW

**Comment by rabin-io** [comment]
Looks good, looking forward to it be merged.
Are the new images on quay are build from the new Containerfile?

**Review by ebenahar — APPROVED** [review]

File: Docker_files/ocsci_container/README.md [hunk 1]
```diff
@@ -7,14 +7,12 @@ The containers are expected to run with a service account that has admin credent
 ### Build Image
 * Build Image on your local machine
 * PARAMS:
-  * BRANCH - branch name [e.g: release-4.12]
   * IMAGE_NAME - image name [e.g. quay.io/ocsci/ocs-ci-container:release-4.12]
   * ENGINE - engine name [podman/docker]
 * Example:
   ```
   make build-image \
-  BRANCH=release-4.12 \
```
Comment (petr-balogh) [inline Docker_files/ocsci_container/README.md:16]: How we are now defining which ocs-ci release branch should be used?

**Review by petr-balogh — APPROVED** [review]

File: Docker_files/ocsci_container/README.md (hunk 1 above)
Comment (rabin-io) [reply Docker_files/ocsci_container/README.md:16]: The Containerfile should be on all branches, and for each branch, when there is a merge into it, the CI system will check out the code, and build the image for this specific branch.

Comment (petr-balogh) [reply Docker_files/ocsci_container/README.md:16]: > The Containerfile should be on all branches, and for each branch, when there is a merge into it, the CI system will check out the code, and build the image for this specific branch.

Yest, it was cleared out in our discussion, I just was not sure when I asked the question earlier than our meeting. I remembered the old use case of git clone and use particular branch and I was not sure how it is handled now.

Thanks

**Comment by petr-balogh** [comment]
@ebenahar can you please re-approve?

**Review by ebenahar — APPROVED** [review]

**Pull request merged**